### DISCUSSION TITLE
Companion 2.8.4 32 bit may have virus?

### DISCUSSION THREAD

**Discussion by Dave-thirteen** [EdgeTX/edgetx#3627] [General]
Tried to download and AVG reports a virus.

Any ideas if its so, or is AVG being picky?

Thanks
Dave.
![comp](https://github.com/EdgeTX/edgetx/assets/112688258/b7981e8c-2e36-48b2-8433-9cef24068179)

**Comment by pfeerick** [comment]
Should be a false positive. I scanned the release files between downloading them from the build server, and uploading them for release, and VirusTotal only raised concern from 1 of the 67 security vendors - [view the report for the installer here](https://www.virustotal.com/gui/file/cfcf6e5e5a9aa01eacc1b18010b860cf1f0cad2216b3fac615b4c22de426fa31) and [the report for the zip file and installer here](https://www.virustotal.com/gui/file/d96cbb2756eb50343498cdd49173d6835361f3740c4e04ed04d0b5b1483dda86/behavior). You can upload the file to https://www.virustotal.com/ and check for yourself - it should pretty much jump to the report page instantly if the files you've downloaded are the same as the original release files since they've already been scanned.

**Comment by Dave-thirteen** [comment]
Peter, thanks for checking for me.
I can't complete the download as my AVG aborts it before it finishes.
Kind regards
Dave.
